matrix_sdk_base/src/models/room.rs

```diff
@@ -471,63 +485,44 @@ impl Room {
     /// disambiguating should be considered a current member of the room.
     ///
     /// Returns a map from MXID to disambiguated name.
-    fn member_disambiguations(
+    fn disambiguation_updates(
         &self,
-        member: &RoomMember,
-        inclusive: bool,
-    ) -> HashMap<UserId, String> {
-        let users_with_same_name = self.shares_displayname_with(member, inclusive);
-        let disambiguate_with = |members: Vec<UserId>, f: fn(&RoomMember) -> String| {
-            members
-                .into_iter()
-                .filter_map(|id| {
-                    self.joined_members
-                        .get(&id)
-                        .or_else(|| self.invited_members.get(&id))
-                        .map(f)
-                        .map(|m| (id, m))
-                })
-                .collect::<HashMap<UserId, String>>()
+        member: &UserId,
+        old_name: Option<String>,
+        new_name: Option<String>,
+    ) -> HashMap<UserId, bool> {
+        // Must be called *before* any changes to the model.
```

Comment:
Is this mean to be a docstring for the method? What must be called before any changes to the model? As per the docstring this method doesn't change the model?

Reply:
Yes, this should be in the docstring.

It seems that this is a stale docstring (for an old version of the method) and I somehow misplaced the new one and never committed it. The comment is talking about the fact that `disambiguation_updates` should be called before we actually make any display name changes in our `RoomMember` structures.

Fixed.